README.md

```diff
@@ -172,37 +185,272 @@ batch.get(
 )
 
 batch.get(
-    "daily_active_addresses/santiment",
-    from_date="2018-06-06",
-    to_date="2018-06-10",
+    "transaction_volume/santiment",
+    from_date="2018-06-01",
+    to_date="2018-06-05",
     interval="1d"
 )
 
-[daa1, daa2] = batch.execute()
+[daa, trx_volume] = batch.execute()
 ```
 
 ## Available metrics
 
-Getting all of the metrics as a list is done, using the following code:
+Getting all of the metrics as a list is done using the following code:
 
 ```python
 san.available_metrics()
 ```
 
-Getting all of the metrics for a given slug is achieved with the following code
-
+Getting all of the metrics for a given slug is achieved with the following code:
 
 ```python
 san.available_metrics_for_slug('santiment')
 ```
 
-Below are described some available metrics and are given examples for fetching and for the returned format.
-
-### All Projects
+Below are described the available metrics and are given examples for fetching them.
+
+### Full list of metrics for a single project
+
+> NOTE: When a new metric is added to the API, `san.available_metrics()` will
+> automatically pick it up and it will be accessible with sanpy, but it might
+> take some time to be added to this documentation. The list below might not be
+> full at times.
+
+The suffixes `_<number>y` and `_<number>d` means that the metric is calculated
+only by taken into account the tokens and coins that have moved in the past
+number of years or days.
+
+All these metrics are returned as a Pandas dataframe with two columns - `datetime`
+and float `value`.
+
+All metrics that do not follow the same format are explicitly listed after that.
+
+#### Holder Metrics
```

Comment:
Doesn't that make the README somewhat long?

Reply:
This PR adds 300 lines and removes 1026 lines. Overall the text is much shorter

Reply:
Yeah, that's correct